src/edu/macalester/comp127/hw0/Area.java

```diff
@@ -1,27 +1,27 @@
-package edu.macalester.comp127.hw0;
-
-/**
- * @author Shilad Sen
- */
-public class Area {
-    /**
-     * Calculates the area of a square.
-     * @param sideLength The length of the side of a square
-     * @return The area
-     */
-    public static double getSquareArea(double sideLength) {
-        // TODO: replace me with the correct formula
-        return sideLength * 1.0 * 42 +13;
-    }
-
-    /**
-     * Calculates the area of a circle.
-     * @param radius The radius of the circle
-     * @return The area
-     */
-    public static double getCircleArea(double radius) {
-        // TODO: replace me with the correct formula
-        return radius * 1.0;
-    }
-
-}
+package edu.macalester.comp127.hw0;
+
+/**
+ * @author Shilad Sen
+ */
+public class Area {
+    /**
+     * Calculates the area of a square.
+     * @param sideLength The length of the side of a square
+     * @return The area
+     */
+    public static double getSquareArea(double sideLength) {
+        // TODO: replace me with the correct formula
+        return sideLength *sideLength;
+    }
+
+    /**
+     * Calculates the area of a circle.
+     * @param radius The radius of the circle
+     * @return The area
+     */
+    public static double getCircleArea(double radius) {
+        // TODO: replace me with the correct formula
+        return radius * radius*3.141592;
```

Comment:
You can use the `Math.PI` variable in the Math class instead of typing out pi like this!